README.md

```diff
@@ -1,3 +1,280 @@
 # sf-functions-python
 
-Python support for Salesforce Functions.
+Python support for [Salesforce Functions](https://developer.salesforce.com/docs/platform/functions/overview).
+
+> Note: This feature is in beta and has been released early so we can collect feedback. It may contain significant problems, undergo major changes, or be discontinued. The use of this feature is governed by the Salesforce.com Program Agreement.
+
+---
+
+# Getting Started with Python for Functions
+
+## Prerequisites
+
+> Any install commands that follow assume a Mac OSX system with Homebrew available.  If you’re on another OS you’ll have to click through the links to get alternative install instructions.
+
+### [Install Python 3](https://www.python.org/downloads/)
+
+```sh
+brew install python3
+```
+
+The installed Python version should be at least `3.10.6` or higher  
+
+You can check this with `python3 --version`
+
+> On some machines it’s necessary to run Python and Pip commands using python3 or pip3 which point to the Homebrew-managed Python interpreter versus running python or pip which tends to point at the system installed Python interpreter.  
+
+### [Install Git](https://git-scm.com/downloads)
+
+```sh
+brew install git
+```
+
+The installed Git version should be at least `2.36.0` or higher
+
+You can check this with `git --version`
+
+### [Install / Update the Salesforce CLI](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli_rc.htm)
+
+If you haven’t already installed the Salesforce CLI, follow 
+[these steps](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm). 
+
+If you already have the Salesforce CLI installed, make sure it is updated to the latest release, which contains the Python 
+functions support:
+
+```sh
+sfdx update
+```
+
+This will update both the sfdx and sf commands.  Verify that:
+
+* `sf --version` is `1.57.0` (or higher)
+* `sfdx --version` is `7.180.0` (or higher)
```

Comment:
the install command now states `sfdx update stable-rc` and i'm going to remove the comment on the `sfdx` version since any `sfdx` commands is this doc are already stable.